README.md

```diff
@@ -303,36 +303,36 @@ If there currently isn't an active session, then calling this method does nothin
+This is enough to get statistics about how many customers are affected by exceptions, or how many customers are using your application over time.
 
-There are two different ways that you can provide different user information which are described below.
-Please be aware of any company privacy policies you have when choosing what type of user information you send to Raygun.
+There are two different ways that you can provide different customer information which are described below.
+Please be aware of any company privacy policies you have when choosing what type of customer information you send to Raygun.
 
 ### The User property
 
-If all you need to identify a user is a single string, then you can set the `User` property.
+If all you need to identify a customer is a single string, then you can set the `User` property.
 Note that setting this to null or whitespace will cause the default random GUID descibed above to be used.
 This string can be whatever you like. Below are some common suggestions.
 
 * Identifying information such as name or email address.
-* An id that doesn't reveal any information about the user, but can be looked up in your own systems to find out who the user is.
-* Your own random string if you don't want to use the one Raygun stores in roaming app data. This may however result in unreliable user statistics in Raygun.
+* An id that doesn't reveal any information about the customer, but can be looked up in your own systems to find out who the customer is.
+* Your own random string if you don't want to use the one Raygun stores in roaming app data. This may however result in unreliable customer statistics in Raygun.
 
 ### The UserInfo property
 
-If a single string is not enough to describe the information that you want to log about a user, then you can set the `UserInfo` property.
-Below are the various properties that you can use to describe the user. The Identifier is the only required field, which can be provided through the constructor.
+If a single string is not enough to describe the information that you want to log about a customer, then you can set the `UserInfo` property.
+Below are the various properties that you can use to describe the customer. The Identifier is the only required field, which can be provided through the constructor.
 
-**Identifier** The unique identifier you want to use to identify this user. Suggestions for what you could set this to are listed in the User property section above.
+**Identifier** The unique identifier you want to use to identify this customer. Suggestions for what you could set this to are listed in the User property section above.
 
-**IsAnonymous** A flag indicating whether the user is logged in (or identifiable) or if they are anonymous. An anonymous user still requires an identifier.
+**IsAnonymous** A flag indicating whether the customer is logged in (or identifiable) or if they are anonymous. An anonymous user/customer still requires an identifier.

```

Comment:
Was it intentional to leave a mention of "user" here?

Reply:
Yes, but we can remove this now.  Wasn't too sure to associate anonymous user with customers or just users. But we are showing anonymous users within Customers, therefore, would make sense to be consistent with customer